### ISSUE TITLE
Compiling pgmodeler on CentOS 7

### ISSUE THREAD

**Issue by hg2581** [pgmodeler/pgmodeler#1406]
I would like to get pgmodeler up and running on CentOS 7. I have installed postgresql from the repository which is version 9.2.24. Qmake is version 3.1 and Qt is version 5.9.7. I have not gotten pkg-config to find libpq although I have libpq.so installed in /sr/lib64. There is no libpq.pc.

Are the above compatible with the github version of pgmodeler?

**Comment by rkhaotix** [comment]
Hi ! Do your distro have a postgresql-devel or something like that?

**Comment by i-love-coffee-i-love-tea** [comment]
Hi!

I don't know. I will checkout the head from github as soon as i find
some free time today. Should also be fine, or do you ask for a reason?

**Comment by hg2581** [comment]
Yes, CentOS 7 does have postgresql-devel but it does not seem to contain libpq.pc that your instructions mention?

**Comment by rkhaotix** [comment]
You can try to tweak the pgModeler.pri and hardcode the path to libpq.so as well the PostgreSQL headers by changing PGSQL_INC and PGSQL_LIB.

**Comment by hg2581** [comment]
On March 7, 2020 8:44:31 PM GMT+01:00, "Raphael Araújo e Silva" <notifications@github.com> wrote:
>You can try to tweak the pgModeler.pri and hardcode the path to
>libpq.so as well the PostgreSQL headers by changing PGSQL_INC and
>PGSQL_LIB. 

Ok, will try that but two questions:
- What is libpq.pc?
- Is the current version of pgmodeler compatible with postgreSQL currently available in the CentOS EPEL repo I mentioned?

**Comment by rkhaotix** [comment]
A detailed explanation about what is pkg-config and *.pc files:
https://people.freedesktop.org/~dbn/pkg-config-guide.html

The current pgModeler supports any modern PostgreSQL release form 9.0 to 12

**Comment by hg2581** [comment]
I have not yet tried compiling pgmodeler but trying to get pgmodeler-0.9.0-alpha1 working with postgres 9.2.24. The former is available in the CentOS 7 EPEL repository which means I would not have to compile it. However, I cannot get the connection parameters working: are you aware of any issues the above alpha1 version with this version of postgresql?
